src/core/specter-wallet.ts

```diff
@@ -26,13 +26,11 @@ export class SpecterWallet {
   }
 
   async listWallets() {
```

Comment:
do we need the `async`?

Reply:
Let's clean that up in another PR by defining the return types for methods that return promises